services/galaxytoolshed/galaxytoolshed-base.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import { NotFound, BaseJsonService } from '../index.js'
+
+const orderedInstallableRevisionsSchema = Joi.array()
+  .items(Joi.string())
+  .required()
+
+const repositoryRevisionInstallInfoSchema = Joi.array()
+  .ordered(
+    Joi.object({
+      times_downloaded: Joi.number().required(),
```

Comment:
Just want to both confirm that this field will always be present, even when the download count is zero?

Also, we typically wouldn't require this to be changed, but since there's one last round of cleanup needed in the downloads class, this could be changed to utilized our `nonNegativeInteger` helper from the `validators` module (located in the `services` directory)

Reply:
Right, the schema is updated with `nonNegativeInteger` 480cc22